src/server/pfs/server/driver_new_storage.go

```diff
@@ -339,7 +339,7 @@ func (d *driver) compact(master *work.Master, outputPath string, inputPrefixes [
 func serializeShard(shard *pfs.Shard) (*types.Any, error) {
 	serializedShard, err := proto.Marshal(shard)
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "error serializing shard")
```

Comment:
I think the extra message does not provide any value here. I think we would be better off with a `WithStack`.